sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -227,6 +273,11 @@ def _fit(self, y, X, fh):
         elif self.fit_strategy == "full":
             for param in self.model.parameters():
                 param.requires_grad = True
+        elif self.fit_strategy == "peft":
```

Comment:
there should be a user facing message here, i.e., they need peft installed

Reply:
updated in code and docstring

Reply:
nice. The error message does not seem informative though.

Reply:
just to confirm, this is the current error message
```
ModuleNotFoundError: No module named 'peft'. 'peft' is a soft dependency and not included in the base sktime installation. Please run: `pip install peft` to install the peft package. To install all soft dependencies, run: `pip install sktime[all_extras]`
```
and I'll improve this

Reply:
Informative errors should contain:

* source location of error - the class
* what the problem is
* how to remedy it